OpenTreeMap/otm-core
# Fix UDF choice editing helper function

## Overview

In the case where choice fields contain numeric values, this helper is called with a number and fails unless we force the value to a string.

Connects https://github.com/OpenTreeMap/otm-clients/issues/441

## Testing Instructions

* Checkout the `develop` branch
* Browse http://localhost:6060/create/ and create an instance called 'numeric-choices'
* Visit http://localhost:6060/numeric-choices/management/udfs/ and click `+ Add Field`
* Create a choice field like this and click `Finish` to save it.
<img width="901" alt="screen shot 2018-11-02 at 10 35 46 am" src="https://user-images.githubusercontent.com/17363/47931116-1c373280-de8b-11e8-883f-8ccc328303b5.png">
* Refresh the page and click `Edit`. Start typing a new choice value and you will see an exception in the console
<img width="1105" alt="screen shot 2018-11-02 at 10 38 45 am" src="https://user-images.githubusercontent.com/17363/47931289-a2537900-de8b-11e8-8482-15cc38bd68da.png">
* Checkout this branch, reload the page, and click `Edit`.
* Verify that you can successfully add a new choice value and save it.
